Tasks/UniversalPackagesV0/universalpublish.ts

```diff
@@ -121,6 +122,7 @@ export async function run(artifactToolPath: string): Promise<void> {
             accountUrl: serviceUri,
             packageName,
             packageVersion: version,
+            publishedPackageVar,
```

Comment:
Instead of passing this in, then conditionally checking for success, can you just setResult around line 130 where it is already tl.setResult(tl.Taskresult.Succeeded...)?  would cut down on changes and unify the success handling.

Reply:
Moved the code